### PR TITLE
[IMP]10701 add module hr_payroll_report_horizontal_xls

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: hr_payroll_report_horizontal_xls/README.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (report_xls, hr_payroll)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline hr_payroll_report_horizontal_xls/README.rst]: Update the URL and use newest template

File: hr_payroll_report_horizontal_xls/README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Maintainer
+----------
+
+.. image:: https://www.elico-corp.com/logo.png
+:alt: Elico Corp
+:target: https://www.elico-corp.com
+
+This module is maintained by Elico Corporation.
+
+Elico Corporation offers consulting services to implement open source management software in SMEs, with a strong involvement in quality of service.
```
Comment (elicoidal) [inline hr_payroll_report_horizontal_xls/README.rst]: newest template

File: hr_payroll_report_horizontal_xls/report_hr_payroll.py [hunk 3]
```diff
@@ -0,0 +1,253 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from datetime import datetime
+
+import xlwt
```
Comment (elicoidal) [inline hr_payroll_report_horizontal_xls/report_hr_payroll.py]: Should protect the import

**Review by elicoidal — CHANGES_REQUESTED** [review]
push this module to OCA/l10n-china

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

**Comment by elicoidal** [comment]
LGTM but travis is failing. is it related?

**Comment by noahzaozao** [comment]
@elicoidal I'll fix it then merge it

**Pull request merged**